packages/fxa-settings/src/components/Settings/SettingsSubRow/index.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
```

Comment:
Nit: missing license

Just wondering, why `SettingsSubRow`? I think it's fine but we have `UnitRow` so eh, could go just `SubRow`?